frontend/benefit/applicant/src/components/applications/Applications.sc.ts

```diff
@@ -49,3 +49,24 @@ export const $SpinnerContainer = styled.div`
 `;
 
 export const $Empty = styled.div``;
+
+export const $AskemContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  min-height: 129px;
+  margin: 0;
+  padding-left: ${(props) => props.theme.spacing.l};
+  padding-right: ${(props) => props.theme.spacing.l};
+  ${respondAbove('sm')`
+    flex-direction: row;
+
```

Comment:
Added empty row before `${respondAbove('sm')` and empty row removed after `flex-direction: row;` would be more readable.

Reply:
I will try to improve it. Just copied this from the `NotificationView` component, as I needed the same layout here.